package.json

```diff
@@ -22,7 +23,9 @@
   ],
   "dependencies": {
     "axios": "^0.21.0",
+    "browserify": "^17.0.0",
     "core-js": "^3.7.0",
+    "tsify": "^5.0.2",
```

Comment:
This should only a dev dependency, right?

Reply:
if you are making an npm then yes correct.